pmd-apex/src/test/java/net/sourceforge/pmd/lang/apex/ast/ApexParserTestBase.java

```diff
@@ -11,15 +11,15 @@ public class ApexParserTestBase {
     protected final ApexParsingHelper apex = ApexParsingHelper.DEFAULT.withResourceContext(getClass());
 
 
-    protected ApexNode<Compilation> parse(String code) {
+    protected ApexRootNode<Compilation> parse(String code) {
```

Comment:
`ApexRootNode<Compilation>` is not a proper type (should be `? extends Compilation`, or just `?` which is the same). Since you changed the call sites, maybe you can change this as well? I had kept it that way to avoid upating the call sites